src/ansible_sign/cli.py

```diff
@@ -386,7 +348,6 @@ def gpg_sign(args):
     if args.prompt_passphrase:
         passphrase = getpass.getpass("GPG Key Passphrase: ")
 
-    signature_output = os.path.join(args.project_root, args.signature_output)
```

Comment:
In theory, line 353 should be changed to use this line to be consistent with the manifest_output logic above.

In practice, I think let's (in a different PR) just drop these arguments completely. Tooling is going to expect the manifest and signature to live in `<project_root>/.ansible-sign/` anyway. There is no need to customize these output paths.

I'll open a PR for that soon (probably tomorrow).